static/scenes/railroad/js/level.js

```diff
@@ -117,11 +117,26 @@ class Level {
       }
       window.santaApp.fire('sound-trigger', 'bl_score_red');
 
-      this.scoreboard.addScore(100);
+      const score = 100;
+      await this.showScoreText(targetObject, score);
+      this.scoreboard.addScore(score);
       // Start the train if it isn't already moving. Ok to call this multiple times.
       this.startTrain();
     }
   }
+  
+  async showScoreText(targetObject, score) {
```

Comment:
It's possible for presents to land at a pretty similar time, and it would be nice to have two or more score text elements on the screen at the time -- could you create a new element rather than reusing the same element? (But we'd also need remove the extra elements when done too)